change/@azure-msal-angular-e959400e-9863-454e-ad9b-aaf61716280e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "feat: adding the metadata npm commands",
```

Comment:
Angular doesn't appear to have any changes. Let's remove the package-lock changes if they're not relevant to this PR

Reply:
I have removed the package-lock.json file and the associated change file